.drone.yml

```diff
@@ -8,7 +8,7 @@ clone:
   clone-rchain:
     image: plugins/git
     commands:
-      - git clone $${RCHAIN_REPO:-https://github.com/rchain/rchain.git} /workdir/rchain && cd /workdir/rchain
```

Comment:
I don't think this should go into master

Reply:
I in inadvertently pushed my local changes.  PTLA